torchtune/rlhf/loss/dpo.py

```diff
@@ -85,15 +80,15 @@ def forward(
         )
 
         chosen_rewards = (
-            self.beta * (policy_chosen_logps - reference_chosen_logps).detach()
+            self.beta * (policy_inputs.chosen_logps - reference_inputs.chosen_logps).detach()
         )
         rejected_rewards = (
-            self.beta * (policy_rejected_logps - reference_rejected_logps).detach()
+            self.beta * (policy_inputs.rejected_logps - reference_inputs.rejected_logps).detach()
         )
 
         return losses, chosen_rewards, rejected_rewards
 
-
+@deprecated(msg="RSOLoss will be deprecated in an upcoming release.")
```

Comment:
Why are we marking as deprecated? Seems this will not be usable with the current changes anyways?